spark/src/main/scala/org/apache/spark/sql/delta/commands/CreateDeltaTableCommand.scala

```diff
@@ -240,11 +240,13 @@ case class CreateDeltaTableCommand(
     updateCatalog(sparkSession, tableWithLocation, postCommitSnapshot, didNotChangeMetadata)
 
 
-    if (UniversalFormat.icebergEnabled(postCommitSnapshot.metadata)) {
+    if (UniversalFormat.icebergEnabled(postCommitSnapshot.metadata) &&
+        !txnUsedForCommit.containsPostCommitHook(IcebergConverterHook)) {
```

Comment:
nit: why don't we use `isReplace` as a condition here?

Reply:
I think this way should be a little more future proof if the condition in the other place changed..